librz/core/cmd/cmd_print.c

```diff
@@ -7288,4 +7224,58 @@ RZ_IPI RzCmdStatus rz_cmd_base64_decode_handler(RzCore *core, int argc, const ch
 	rz_cons_println((const char *)buf);
 	free(buf);
 	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_print_bitstream_handler(RzCore *core, int argc, const char **argv, RzOutputMode mode) {
+	st32 len = (st32)rz_num_math(core->num, argv[1]), skip = 0;
+	if (!rz_str_isnumber(argv[1]) || len < 0) {
+		RZ_LOG_ERROR("len is not a positive number\n");
+		return RZ_CMD_STATUS_ERROR;
+	}
+	if (argv[2]) {
+		skip = (st32)rz_num_math(core->num, argv[2]);
+		if (!rz_str_isnumber(argv[2]) || skip < 0) {
+			RZ_LOG_ERROR("skip is not a positive number\n");
+			return RZ_CMD_STATUS_ERROR;
+		}
+	}
+	// `pb len skip` means skip <skip> bits then print <len> bits
+	char *buf = calloc(1, len + skip + 1);
+	if (!buf) {
+		RZ_LOG_ERROR("Fail to allocate memory\n");
+		return RZ_CMD_STATUS_ERROR;
+	}
+	rz_str_bits(buf, core->block, len + skip, NULL);
+	if (!RZ_STR_ISEMPTY(buf + skip)) {
+		rz_cons_println(buf + skip);
+	}
```

Comment:
I would always print the result, without the `if`

Reply:
This will break the test `db/cmd/cmd_0 pB 0`. Or I alter the test to satisfy the code that prints directly?

Reply:
pB 0 i expect an empty line just with a newline, right? If so, yes adjust the test it seems fine.